src/database/types.ts

```diff
@@ -60,6 +60,11 @@ export type CreateCollectionInput = {
   title: string;
 };
 
+export type UpdateLabelInput = {
+  externalId: string;
+  name: string;
```

Comment:
I think `name` should be a required prop here.

Reply:
@Herraj not sure what do you mean. I think it is already implied it is required in the `UpdateLabelInput` type?

Reply:
Yep, you're right, it is required!